data/json/furniture_and_terrain/appliances.json

```diff
@@ -474,6 +474,31 @@
       { "item": "steel_plate", "count": [ 2, 4 ] }
     ]
   },
+  {
+    "type": "vehicle_part",
+    "id": "ap_portable_ac",
+    "name": { "str": "portable air conditioner" },
+    "categories": [ "utility" ],
+    "description": "A modified vehicle-mounted area cooler.  A custom made casing with a hose attached which allows the exhaust heat to be directed out of the room.",
```

Comment:
It would make much more sense for this to just be a regular furniture that you can find in store and houses, just like irl, instead of being weird makeshift vehicle thing.